tests_random/utils.py

```diff
@@ -76,6 +76,20 @@ def assert_equals(frame1, frame2):
         assert same_iterables(frame1.to_list(), frame2.to_list())
 
 
+def log(fn):
```

Comment:
May be use a name different from `math.log`?

Reply:
sure